src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -57,6 +57,7 @@ export type Layers = {
   readonly overriddenInfobox?: OverriddenInfobox;
   readonly overriddenProperties?: { [id: string]: any };
   /** Selects the layer with the specified ID; if the ID is undefined, the currently selected later will be deselected. */
+  readonly layersInViewport: () => Layer[];
```

Comment:
The plugin API often uses getters, so make sure to return the value directly instead of returning a function.

```suggestion
  readonly layersInViewport: Layer[];
```